README.md

```diff
@@ -5,7 +5,7 @@
 - [Notion](https://www.notion.so/pesto/Batch-11-project-specs-40cef8d675c04ac0b72a7c2995073781)
 
 ### Instructions
-- Please create a issue first and then attach that issue to the PR, so that every one know what’s the need of the PR. Attaching issues to the PR using github keywords, will close the issues once the PR is merged. More info [here]((https://help.github.com/en/articles/closing-issues-using-keywords)).
+- Please create a issue first and then attach that issue to the PR, so that every one know what’s the need of the PR. Attaching issues to the PR using github keywords, will close the issues once the PR is merged. More info [here](https://help.github.com/en/articles/closing-issues-using-keywords).
```

Comment:
This is already merged into master right? If yes please rebase it.

Reply:
Done